theano/gpuarray/opt.py

```diff
@@ -264,19 +264,26 @@ def __init__(self, local_optimizers_all, local_optimizers_map):
         self.local_optimizers_all = local_optimizers_all
         self.local_optimizers_map = local_optimizers_map
         self.failure_callback = None
+        self.new_opts = []
 
     def add_requirements(self, fgraph):
         fgraph.attach_feature(toolbox.ReplaceValidate())
 
+    def get_local_optimizers(self):
```

Comment:
Yeah, forgot to remove. When i added this, i forgot about the list that we maintain for local opts